src/main/java/com/brandwatch/internship/fetchandstoretweets/services/CrawlerService.java

```diff
@@ -0,0 +1,41 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.MentionsRepository;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import com.brandwatch.internship.fetchandstoretweets.services.utility.TweetsToMentions;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.social.twitter.api.Tweet;
+import org.springframework.social.twitter.api.Twitter;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CrawlerService {
```

Comment:
Consider naming this class `CrawlerJob` to specify that the only purpose of this class is to periodically run the job you defined. 
Also `@Service` may be changed to `@Component`